assets/scss/_custom.scss

```diff
@@ -154,6 +154,18 @@ body.td-404 main .error-details {
   }
 }
 
+//Arabic language
+body:lang(ar) {
+  .navbar-brand {
+    left: 16px;
+  }
+  .announcement-main {
+
+    direction: ltr;
+    text-align: left;
+
+  }
+}
```

Comment:
- This change belongs in its own PR.
- Ideally, make it work for all LTR languages and not just Arabic.

SIG Docs can help you make and land that PR in _main_ so that the Arabic work can build upon it.

Reply:
I'm happy to support the work to complete LTR support; I don't plan to work on it all myself.

Reply:
We should definitely separate it out so it can land as its own change, and into _main_.

Reply:
I think should be fixed in https://github.com/google/docsy